tao_compiler/mlir/xla/ral/context/custom_library/transpose_gpu.cu.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "tensorflow/compiler/mlir/xla/ral/context/custom_library/transpose.h"
+#include "tensorflow/core/kernels/conv_2d_gpu.h"
+#include "tensorflow/core/lib/math/math_util.h"
```

Comment:
better to not introduce any TF dependency in RAL whenever possible.

Reply:
done